src/main/kotlin/no/nav/syfo/kafka/producers/dittsykefravaer/DittSykefravaerMeldingKafkaProducer.kt

```diff
@@ -0,0 +1,58 @@
+package no.nav.syfo.kafka.producers.dittsykefravaer
+
+import no.nav.syfo.Environment
+import no.nav.syfo.kafka.common.JacksonKafkaSerializer
+import no.nav.syfo.kafka.common.producerProperties
+import no.nav.syfo.kafka.common.topicDittSykefravaerMelding
+import no.nav.syfo.kafka.producers.dittsykefravaer.domain.MeldingKafkaDto
+import no.nav.syfo.kafka.producers.dittsykefravaer.domain.OpprettMelding
+import no.nav.syfo.kafka.producers.dittsykefravaer.domain.Variant
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerConfig
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.slf4j.LoggerFactory
+import java.time.LocalDate
+import java.time.ZoneOffset
+import java.util.UUID
+
+class DittSykefravaerMeldingKafkaProducer(
+    val env: Environment
+) {
+    private val log = LoggerFactory.getLogger("no.nav.syfo.kafka.producers.dittsykefravaer.DittSykefravaerMeldingKafkaProducer")
+    private val kafkaConfig = producerProperties(env).apply {
+        put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, "org.apache.kafka.common.serialization.StringSerializer")
+        put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, JacksonKafkaSerializer::class.java)
+    }
+    private val kafkaProducer = KafkaProducer<String, MeldingKafkaDto>(kafkaConfig)
+
+    fun sendMelding(fnr: String, tilfelleEndDate: LocalDate) {
+        val uuid = "esyfovarsel-${UUID.randomUUID()}"
+        val melding = MeldingKafkaDto(
+            OpprettMelding(
+                oppgave39UkerTekst,
```

Comment:
Denne produceren bør kunne brukes til alle type varsel som skal sendes til Ditt sykefravær, så vi bør ikke ha referanse til varselspesifikke, i dette tilfelle 39uker, tekster etc

Reply:
Hvis det føles naturlig og passer inn, så kanskje dette kan være en god anledning til å lage
```
class MerVeiledningVarselService(
    val senderFacade: SenderFacade
) {
    fun sendVarselTilArbeidstaker(varselHendelse: ArbeidstakerHendelse) {
        // Some useful code
    }
}
```

Reply:
@kaja-nav Jeg liker tanken, men `MotebehovVarselService` og `OppfolgingsplanVarselService` brukes vel for varselbus-varsler? Tenker du på å refaktorere `SendVarselService` for planlagte varsler? E.g her: https://github.com/navikt/esyfovarsel/blob/master/src/main/kotlin/no/nav/syfo/service/SendVarselService.kt#L69 ?